test/e2e/specs/site-editor/settings-sidebar.spec.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Settings sidebar', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.activateTheme( 'emptytheme' ),
+			requestUtils.deleteAllTemplates( 'wp_template' ),
+			requestUtils.deleteAllTemplates( 'wp_template_part' ),
+		] );
+	} );
+
+	test.beforeEach( async ( { admin } ) => {
+		await admin.visitSiteEditor( {
+			postId: 'emptytheme//index',
+			postType: 'wp_template',
+			canvas: 'edit',
+		} );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.activateTheme( 'twentytwentyone' ),
+			requestUtils.deleteAllTemplates( 'wp_template' ),
+			requestUtils.deleteAllTemplates( 'wp_template_part' ),
+		] );
+	} );
+
+	test.describe( 'Template tab', () => {
+		test( 'should open template tab by default if no block is selected', async ( {
+			editor,
+			page,
+		} ) => {
+			await editor.openDocumentSettingsSidebar();
+
+			await expect(
+				page
+					.getByRole( 'region', { name: 'Editor settings' } )
+					.getByRole( 'button', { name: 'Template (selected)' } )
+			).toHaveClass( /is-active/ );
```

Comment:
I think we can improve active tab assertions after #56959 is merged.

Untested example:

```js
await expect(
		page
			.getByRole( 'region', { name: 'Editor settings' } )
			.getByRole( 'tab', { selected: true } )
	).toHaveText( 'Template' );
```

Reply:
The active tab assertions work like a charm - https://github.com/WordPress/gutenberg/pull/57446/files.